docs/standard/garbage-collection/fundamentals.md

```diff
@@ -150,115 +153,125 @@ Before a garbage collection starts, all managed threads are suspended except for
 
 The following illustration shows a thread that triggers a garbage collection and causes the other threads to be suspended.
 
-![When a thread triggers a Garbage Collection](../../../docs/standard/garbage-collection/media/gc-triggered.png "When a thread triggers a Garbage Collection")
+![When a thread triggers a Garbage Collection](./media/gc-triggered.png)
 
-## Manipulating unmanaged resources
+## Manipulate unmanaged resources
 
-If your managed objects reference unmanaged objects by using their native file handles, you have to explicitly free the unmanaged objects, because the garbage collector tracks memory only on the managed heap.
+If managed objects reference unmanaged objects by using their native file handles, you have to explicitly free the unmanaged objects, because the garbage collector only tracks memory on the managed heap.
 
-Users of your managed object may not dispose the native resources used by the object. To perform the cleanup, you can make your managed object finalizable. Finalization consists of cleanup actions that you execute when the object is no longer in use. When your managed object dies, it performs cleanup actions that are specified in its finalizer method.
+Users of the managed object may not dispose the native resources used by the object. To perform the cleanup, you can make the managed object finalizable. Finalization consists of cleanup actions that execute when the object is no longer in use. When the managed object dies, it performs cleanup actions that are specified in its finalizer method.
 
 When a finalizable object is discovered to be dead, its finalizer is put in a queue so that its cleanup actions are executed, but the object itself is promoted to the next generation. Therefore, you have to wait until the next garbage collection that occurs on that generation (which is not necessarily the next garbage collection) to determine whether the object has been reclaimed.
 
+For more information about finalization, see <xref:System.Object.Finalize?displayProperty=nameWithType>.
+
 ## Workstation and server garbage collection
 
 The garbage collector is self-tuning and can work in a wide variety of scenarios. You can use a configuration file setting to set the type of garbage collection based on the characteristics of the workload. The CLR provides the following types of garbage collection:
 
-- Workstation garbage collection, which is for all client workstations and stand-alone PCs. This is the default setting for the [\<gcServer> element](../../../docs/framework/configure-apps/file-schema/runtime/gcserver-element.md) in the runtime configuration schema.
+- Workstation garbage collection (GC), which is for all client workstations, stand-alone PCs, and mobile apps built using Xamarin. Workstation GC is the default setting for the [gcServer element](../../../docs/framework/configure-apps/file-schema/runtime/gcserver-element.md) in the run-time configuration schema.
```

Comment:
@Maoni0 Please verify statement about mobile apps and workstation gc.

Reply:
I don't think we should mention Xamarin here because it's a different runtime. 

I would say something like "Workstation garbage collection is designed for client apps and is the default of the GC flavor for stand alone apps. for hosted apps (like by asp.net) the default is decided by the host". 

I'm not sure what "This is the default setting for the gcServer element in the runtime configuration schema." means - should probably just delete it or make it more explicit by say "this means the default value for gcServer in the config is false"